magnum-pg/src/main/scala/com/augustnagro/magnum/pg/json/JsonBDbCodec.scala

```diff
@@ -0,0 +1,30 @@
+package com.augustnagro.magnum.pg.json
+
+import com.augustnagro.magnum.DbCodec
+import org.postgresql.util.PGobject
+
+import java.sql.{PreparedStatement, ResultSet, Types}
+
+trait JsonBDbCodec[A] extends DbCodec[A]:
+
+  def encode(a: A): String
+
+  def decode(json: String): A
+
+  override def queryRepr: String = "?"
+
+  override val cols: IArray[Int] = IArray(Types.OTHER)
```

Comment:
Why `Types.OTHER` here? 🤔
I implemented this way: `IArray(Types.JAVA_OBJECT)`. Not sure to remember why. Probably something I found on internet

Reply:
The PgCodecTests "insert MagCar" test fails when using JAVA_OBJECT but works with OTHER

```
INSERT INTO mag_car (id, text_colors, text_color_map, last_service, my_json_b, my_xml) VALUES (?, ?, ?, ?, ?, ?)
With message:
org.postgresql.util.PSQLException: Unknown Types value.
```